lib/streamlit/forward_msg_queue.py

```diff
@@ -51,59 +53,47 @@ def get_debug(self) -> Dict[str, Any]:
             "ids": list(self._delta_index_map.keys()),
         }
 
-    def __iter__(self) -> Iterator[ForwardMsg]:
-        return iter(self._queue)
-
```

Comment:
This was never safe in our multi-threaded queue world - but thankfully it was never used, either